.github/workflows/api-rewrite-cassettes.yml

```diff
@@ -36,8 +36,58 @@ jobs:
           PROOF_API_TOKEN_DEV: "op://PROOF/PROOF_Test_User/dev token"
           PATH_ROOTS: "op://PROOF/PROOF_PATH_ROOTS/credential"
 
-      - name: Run tests
-        run: make test_api_rewrite
+      - name: Run tests and process changes
+        id: process-changes
+        run: |
+          make test_api_rewrite_json_report || true
```

Comment:
Love the elegance of `make [command] || true`

Reply:
yeah, there's many ways to achieve this but this one seems good